app/src/main/java/com/sanaa/tudee_assistant/presentation/component/button/TextButton.kt

```diff
@@ -0,0 +1,126 @@
+package com.sanaa.tudee_assistant.presentation.component.button
+
+import android.graphics.Color
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sanaa.tudee_assistant.presentation.component.button.utils.ButtonContent
+import com.sanaa.tudee_assistant.presentation.component.button.utils.SpinnerIcon
+import com.sanaa.tudee_assistant.presentation.design_system.theme.Theme
+import com.sanaa.tudee_assistant.presentation.design_system.theme.TudeeTheme
+
+
+@Composable
+fun TextButton(
+    modifier: Modifier = Modifier,
+    label: String,
+    enabled: Boolean = true,
+    isLoading: Boolean = false,
+    onClick: () -> Unit = {}
+) {
+    Row (
+        modifier
+            .clickable(
+                enabled = enabled,
+                onClick = onClick
+            ),
+        horizontalArrangement = Arrangement.spacedBy(8.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ){
+        ButtonContent(label, enabled, isLoading)
+    }
+}
+
+
+
+
+
+
+
+@Preview(showBackground = true)
+@Composable
+fun TextButtonLight(modifier: Modifier = Modifier) {
```

Comment:
Since the only difference between light and dark preview is a boolean flag, please avoid code duplication by using `PreviewParameter` to create a reusable theme preview method.

search about PreviewParameter

Reply:
@mostafa-n3ma  try to use this `@PreviewLightDark`